File: JAVA/Fibonacci.java
import java.util.*;
public class Fibonacci 
{
    public static void main(String args[])
    {
        System.out.print("Enter the number of terms of series to print: ");
        Scanner s = new Scanner(System.in);
        int n = s.nextInt();
        for(int i = 0;i<n;i++)
        {
            System.out.print(fibonacci(i)+" ");
        }
    }

    public static int fibonacci(int n)
    {
        if(n == 0 || n == 1)
        {
            return n;
        }
        if(n == 2)
        {
            return 1;
        }
        return fibonacci(n-1)+fibonacci(n-2);
    }
}

File: C/Fibonacci.c
#include<stdio.h>
int main()
{
    int a,b,first=0,second=1,d;
    printf("Enter the number of terms of fibonacci series: ");
    scanf("%d",&a);
   for(b=0;b<a;b++)
   {
     printf("%d ",first);
     d=first+second;
     first=second;
     second=d;
   }
}